tools/wa_user_directory/config.yaml

```diff
@@ -4,6 +4,25 @@ device: generic_android
 trace-cmd:
   buffer_size: 102400
   report: false
+  events: [
+    "sched_switch",
+    "sched_wakeup",
+    "sched_wakeup_new",
+    "sched_overutilized",
+    "sched_load_avg_cpu",
```

Comment:
These are old version format events... can we run on kernels where we use the new set of events for PELT signals?

Reply:
Yeah we can add those, I'll align this with what is done in the PELT tests.

Reply:
IMO we could just list the old & new ones.